pkg/operator/ceph/csi/spec.go

```diff
@@ -428,3 +433,65 @@ func createCSIDriverInfo(clientset kubernetes.Interface, name string) error {
 
 	return err
 }
+
+// ValidateCSIVersion checks if the configured ceph-csi image is supported
+func ValidateCSIVersion(clientset kubernetes.Interface, namespace, csiPluginImage, rookImage, serviceAccountName string) error {
+	// skip check if running unsupported images is allowed
+	if AllowUnsupported {
```

Comment:
With the other comment below, we would always need to run the version check to verify that the minimum version is met.

Reply:
See my comment above.